azure-tools-common/src/main/java/com/microsoft/azure/common/function/handlers/runtime/WindowsFunctionRuntimeHandler.java

```diff
@@ -59,11 +66,25 @@ protected WindowsFunctionRuntimeHandler(Builder builder) {
                 appWithCreate = appWithExistingServicePlan.withExistingResourceGroup(resourceGroup);
             }
         }
+        appWithCreate.withJavaVersion(javaVersion).withWebContainer(null);
         return appWithCreate;
     }
 
     @Override
     public FunctionApp.Update updateAppRuntime(FunctionApp app) {
-        return app.update();
+        final JavaVersion serverJavaVersion = app.javaVersion();
```

Comment:
when local configuration has not configured java version and shall we update Java version at server side?  

```
if (javaVersion.equals(serverJavaVersion)) {
            Log.info(String.format(HOST_JAVA_VERSION, serverJavaVersion));
        } else if (JavaVersion.OFF.equals(serverJavaVersion)) {
            Log.info(HOST_JAVA_VERSION_OFF);
            update.withJavaVersion(javaVersion);
        }
```

Reply:
Will update the server side java version only if user set javaVersion in `pom.xml`
``` java
else if (StringUtils.isNotEmpty(runtimeConfiguration.getJavaVersion())) {
            // Will update function host java version if user specify it in pom
            Log.info(String.format(HOST_JAVA_VERSION_UPDATE, serverJavaVersion, javaVersion));
            update.withJavaVersion(javaVersion);
        }
```